code/modules/admin/admin.dm

```diff
@@ -154,14 +154,18 @@ proc/admin_notice(var/message, var/rights)
 				<A href='?src=\ref[src];simplemake=human;species=Xenomorph Queen;mob=\ref[M]'>Queen</A> \] |
 				\[ Crew: <A href='?src=\ref[src];simplemake=human;mob=\ref[M]'>Human</A>
 				<A href='?src=\ref[src];simplemake=human;species=Unathi;mob=\ref[M]'>Unathi</A>
-				<A href='?src=\ref[src];simplemake=human;species=Tajaran;mob=\ref[M]'>Tajaran</A>
+				<A href='?src=\ref[src];simplemake=human;species=Tajara;mob=\ref[M]'>Tajaran</A>

```

Comment:
Could this entire thing not be refactored to just scroll through all species datums and add them to the list...? Seems like it'd help prevent future things missing. If necessary, add a flag to the species datum which determines if it's admin spawnable or not.

Reply:
Addressed